migrations/migration_resource.py

```diff
@@ -0,0 +1,37 @@
+"""Migration Module
+
+Captures common methods needed by the system for migrations, by specializing
+the `fhirclient.Basic` class.
+"""
+from fhirclient.models.basic import Basic
+
+from isacc_messaging.models.fhir import (
+    HAPI_request,
+)
+
+class MigrationManager(Basic):
+    def __init__(self, jsondict=None, strict=True):
+        super(Basic, self).__init__(jsondict=jsondict, strict=strict)
+
+    def __repr__(self):
+        return f"{self.resource_type}/{self.id}"
+
+    def get_latest_migration():
+        return 0
```

Comment:
I imagine this is just a place holder, but again, I'd return `None` if there isn't one already persisted.